src/modules/systemlib/printload.c

```diff
@@ -324,14 +282,58 @@ void print_load_buffer(uint64_t t, char *buffer, int buffer_length, print_load_c
 		cb(user);
 	}
 
+	// Print footer
+	buffer[0] = 0;
+	cb(user);
+	float task_load;
+	float sched_load;
+
+	task_load = (float)(print_state->total_user_time) * print_state->interval_time_ms_inv;
+
+	/* this can happen if one tasks total runtime was not computed
+	   correctly by the scheduler instrumentation TODO */
+	if (task_load > (1.f - idle_load)) {
+		task_load = (1.f - idle_load);
+	}
+
+	sched_load = 1.f - idle_load - task_load;
+
+	snprintf(buffer, buffer_length, "Processes: %d total, %d running, %d sleeping",
+		 system_load.total_count,
+		 print_state->running_count,
+		 print_state->blocked_count);
+	cb(user);
+	snprintf(buffer, buffer_length, "CPU usage: %.2f%% tasks, %.2f%% sched, %.2f%% idle",
```

Comment:
The very first output here when starting `top` is wrong (0% usage), and I think it would be better to skip the entire output block for the first time (as above with `continue; // not enough data yet`).